packages/features/bookings/lib/handleSeats/reschedule/attendee/attendeeRescheduleSeatedBooking.ts

```diff
@@ -76,17 +77,19 @@ const attendeeRescheduleSeatedBooking = async (
     ]);
   }
 
-  const copyEvent = cloneDeep(evt);
-
-  const updateManager = await eventManager.reschedule(copyEvent, rescheduleUid, newTimeSlotBooking.id);
```

Comment:
We were calling the whole `reschedule` logic of the `EventManager` class but we only need to update calendar attendees.